README.md

```diff
@@ -1,153 +1,169 @@
+* **Cluster Type**: Docker [Unified Control Plane](https://www.docker.com/products/docker-universal-control-plane) | Docker [Swarm](https://docs.docker.com/swarm/)
+* **Cluster OS**: [CentOS 7.2](https://www.centos.org/)
+* **Cluster Target**: Bare Metal | [Vagrant](https://www.vagrantup.com/)
 
-##Definitions and Terms:
-###Server/Node/Host/Asset
-A standalone physical/virtual machine with predefined/discovered compute, memory, storage
+##Definitions
+To better understand Cluster Manager, you should know a few system definitions.
+
+###Node
+A physical or virtual server with predefined/discovered compute, memory, storage
 and networking capabilities.
 
 ###Cluster
-A collection of one or more nodes. Nodes may or may not be homogeneous in their capabilities.
+A collection of one or more nodes running clustering software such as Docker Swarm.
+Nodes may or may not be homogeneous in their capabilities.
 
-###Base Image/Image
-A base OS image with *minimal set* of pre-installed packages that are needed for cluster
-management service to manage that node. For instance, for automated discovery of node a
-cluster membership service like Serf needs to be pre-installed in the base OS and started
-at node's bootup.
+###Image
+An operating system with a *minimal set* of pre-installed packages used by a node. For instance,
+node automated discovery requires a cluster membership service like [Serf](https://www.serfdom.io/)
+to be pre-installed and started on the node's operating system during bootstrapping.
 
 ###Bootstrap
-When used in context of node, it refers to the process of installing the node image and
-bringing up the node first time. It is sometime also referred to as node onboarding.
 
-When used in context of cluster, bootstrap refers to the process of adding first bootstrapped
-node to the cluster.
+**Node Bootstrap**: The process of installing the node image and booting the node for the first time.
+
+**Cluster Bootstrap**: The process of adding bootstrapped nodes to the cluster.
 
 ###Lifecycle
-A collection of well-defined states that a node or cluster goes through when certain events happen.
+A collection of well-defined states that a node or cluster transitions through based on system events.
+
+###Infra Service
+A userspace application or a kernel module that runs on a node and provides a support service to
+the clustering system software. Examples include plugins, drivers, or components for networking
+and storage, key/value stores, etc..
 
-###Infra App/Infra Service
-An userspace application or a kernel module that is used to provide one or more system services
-like plugins/drivers or controller components for networking and storage; key/value stores; etc.
+##Features
+
+Cluster Manager provides the following features. Not all features are implemented, or will be implemented in the initial release. Please review the [Roadmap](management/ROADMAP.md) or continue reading to understand individual feature status.
```

Comment:
Thanks for the clarification. To avoid confusion, I am changing from cluster manager (i.e. the service at the heart of contiv cluster), to contiv cluster that refers to the integrated collection of software (i.e. the stack). I consider the integrated stack as clusterctl, custerm, collins, serf, etc..I will remove the others.